presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatistics.java

```diff
@@ -14,15 +14,21 @@
 package com.facebook.presto.spi.statistics;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import org.openjdk.jol.info.ClassLayout;
 
 import java.util.Objects;
 import java.util.Optional;
 
+import static com.facebook.presto.spi.statistics.DoubleRange.RANGE_SIZE;
+import static com.facebook.presto.spi.statistics.Estimate.ESTIMATE_SIZE;
 import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 
 public final class ColumnStatistics
```

Comment:
Can we add some tests that show that the memory estimations are reasonably accurate?

Reply:
A test in the SPI: `TestColumnStatistics` was added to address this